Sources/LeafKit/LeafRenderer.swift

```diff
@@ -215,7 +54,7 @@ public final class LeafRenderer {
         }
     }
 
-    func serialize(_ doc: LeafAST, context: [String: LeafData]) throws -> ByteBuffer {
+    private func serialize(_ doc: LeafAST, context: [String: LeafData]) throws -> ByteBuffer {
```

Comment:
Only other changes are whitespace & explicitly making serialize private instead of internal for consistency